src/Components/Onboarding/Steps/Layout.tsx

```diff
@@ -25,49 +25,40 @@ const Container = styled.div`
 const MainTitle = styled(StyledTitle)`
   text-align: center;
   line-height: normal;
-  margin-bottom: 6px;
+  margin-top: 60px;
+  margin-bottom: 10px;
   ${media.sm`
     text-align: left;
   `};
 `
 const Subtitle = styled(StyledTitle)`
+  ${primary.style};
   color: ${Colors.grayDark};
-  margin-top: 6px;
-  margin-bottom: 30px;
+  margin-top: 0px;
+  margin-bottom: 60px;
   text-align: center;
   line-height: normal;
   ${media.sm`
     text-align: left;
     margin-bottom: 15px;
-    font-size: 20px
+    font-size: 13px;
   `};
 `
 
 const ItemContainer = styled.div`
   padding-bottom: 50px;
 `
 
-/* MS IE11 and Edge don't support for the sticky position property */
 const FixedButttonContainer = styled.div`
   width: 100%;
   position: fixed;
   bottom: 0px;
   left: 0px;
-`
-
-/* Mobile safari doesn't support for the fixed position property:
- *   https://www.eventbrite.com/engineering/mobile-safari-why/
- **/
-const StickyButtonContainer = styled.div`
-  position: sticky;
```

Comment:
Have you tested this on different sizes of iOS devices? They are tricky and don't always respect `position: fixed`.

Reply:
Not sure what's going on with storybooks on Safari rn (lmknow if you're seeing the same and I'll log an issue to keep track of it). But anyway I reverted back to the other implementation with just style updates so we should be covered. 👍